Instruments/config/2017/Skimmer.cfg

```diff
@@ -161,36 +153,45 @@ file: WJetsToLNu_HT-2500ToInf.root
 [VV]
 merged_output: VV.root
 file_xs: 45.99 WWToLNuQQ_NNPDF31.root WWToLNuQQ_NNPDF31_ext1.root
-file_xs: 11.08 WWTo2L2Nu_ext1.root
-file_xs: 47.73 WWTo4Q_NNPDF31.root WWTo4Q_ext1.root
+file_xs: 45.99 WWTo2L2Nu_NNPDF31_PSweights_ext1.root
+file_xs: 47.73 WWTo4Q_NNPDF31.root
+file_xs: 47.73 WWTo4Q_NNPDF31_PSweights_ext1.root
 file_xs: 0.637 WZTo3LNu_0Jets_MLL-50.root
 file_xs: 0.337 WZTo3LNu_1Jets_MLL-50.root
 file_xs: 0.066 WZTo3LNu_2Jets_MLL-50.root
 file_xs: 0.096 WZTo3LNu_3Jets_MLL-50.root
```

Comment:
below there is WZTo3LNu-amcatnloFXFX.root - which is the inclusive W->lnu, Z->ll sample => comment jet-binned samples